src/App/Pages/BaseContentPage.cs

```diff
@@ -130,7 +130,7 @@ protected async Task<AvatarImageSource> GetAvatarImageSourceAsync(bool useCurren
             if (useCurrentActiveAccount)
             {
                 return new AvatarImageSource(await _stateService.GetActiveUserIdAsync(),
-                    await _stateService.GetNameAsync(), await _stateService.GetEmailAsync());
+                    await _stateService.GetNameAsync(), await _stateService.GetEmailAsync(), await _stateService.GetAvatarColorAsync());
```

Comment:
Even though this is just fine, a little while ago I've added `GetActiveUserCustomDataAsync<T>(Func<Account, T> dataMapper)` which can be used when we need multiple data from the account/profile in one call.
So instead of doing three await calls to the state service we can just do one (and also spare the added method `GetAvatarColorAsync()`):
```c#
var t = await GetActiveUserCustomDataAsync<Tuple<string, string, string>>(a => a?.Profile != null ? new Tuple<string, string, string>(a.Profile.Name, a.Profile.Email, a.Profile.AvatarColor) : null);
```
Can't remember well if we can use concise tuples given the framework we are targeting, but if we can we can write it as:
```c#
(string name, string email, string avatarColor) = await GetActiveUserCustomDataAsync(a => a?.Profile != null ? (a.Profile.Name, a.Profile.Email, a.Profile.AvatarColor) : null);
```
Also it could be used with a custom type instead of a `Tuple`

what do you think @mp-bw ?